extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -672,6 +672,36 @@ abstract class RangerSparkExtensionSuite extends AnyFunSuite
     }
     doAs("admin", assert(sql("show tables from global_temp").collect().length == 0))
   }
+
+  test("[KYUUBI #3607] Support {OWNER} variable defined in Ranger Policy") {
+    // For now, only hive catalog is supported
+    assume(catalogImpl == "hive")
```

Comment:
how about move this ut to `HiveCatalogRangerSparkExtensionSuite`?
and remove checking the assume and checking catalogImpl to 'hive'

Reply:
Whether a table supports owner semantic is affected both by `catalogImpl` and by `CatalogExtension` implementation. 
After a second thought, I added `hasTableOwner` field into `SparkSessionProvider`. `hasTableOwner` defaults to true. 
If tables in chid class of `PrivilegesBuilderSuite` and `RangerSparkExtensionSuite` does not have owner, this child class should override `hasTableOwner` field with a `false`.